next-i18next.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    i18n: {
+        defaultLocale: 'pl',
+        locales: ['pl'],
+    },
+    react: { useSuspense: false }
```

Comment:
why do we disable it?

Reply:
Because it cause error: https://stackoverflow.com/questions/67894982/react-i18next-you-will-need-to-pass-in-an-i18next-instance-by-using-initreacti